### ISSUE TITLE
Text Across Multiple Documents

### ISSUE THREAD

**Issue by chantelwetzel-noaa** [nwfsc-assess/sa4ss#6]
I found a way to reference tex or rmd files that are outside the main directory where the document is being rendered by using the LaTeX command \input which will be useful to create area specific assessment documents for a single-species.  However, the bib file is not being located for those files.  Need to update the packages in the  sa4ss.sty.  I tried following the guidance [here](https://tex.stackexchange.com/questions/128017/no-citations-bibliography-processed-produced-when-using-include) but ran into a package conflict with the hyperref package.  Need to dig into this a bit further.

**Comment by kellijohnson-NOAA** [comment]
Thanks @chantelwetzel-noaa for trying things out.

Is the issue that you want to include multiple bib files or the bibliographic references that are in a shared file used via \input are not being rendered?

**Comment by chantelwetzel-noaa** [comment]
In theory we should be able to have one single bib file, but there are issues with LaTeX locating that bib for docs outside the main folder.  This [issue](https://tex.stackexchange.com/questions/192534/how-to-access-files-outside-of-current-direcory) show what I am thinking of in terms of folder structure where.  My [repo](https://github.com/chantelwetzel-noaa/copper_rockfish_2021/tree/master/write_up) also shows an example where the state specific folders with be separate assessment documents but the "general_text" folder will hold any text snippets that will be shared across reports.  As of right now using \input command in LaTeX works but if there are any references cited in the "general_text" folder that are called in the state specific document, LaTeX fails to build due to unexpected references.  I have found some suggestions online (see link in original issue) of how to deal with this problem but when I tested them there were package conflicts.  I can continue to work on this issue but wanted to make an issue so I did not lose track of what I found.
